builder/bundleset.go

```diff
@@ -9,6 +9,8 @@ import (
 	"path/filepath"
 	"regexp"
 	"strings"
+
+	"github.com/clearlinux/mixer-tools/swupd"
```

Comment:
This is out of scope of this PR, but I think the better thing to do would be to break out bundleset.go and bundle_control.go into their own package now, and include it in swupd and builder where needed, instead of pulling all of swupd in just for a header here.